src/BursarExportPlugin.tsx

```diff
@@ -71,6 +71,8 @@ export default function BursarExportPlugin() {
   }
 
   return (
+    <>
+    <TitleManager page={useIntl().formatMessage({ id: "Users settings - Transfer criteria" })}></TitleManager>
```

Comment:
To use `useIntl` in this case, you want to have `const intl = useIntl();` at top of file, below `const stripes = useStripes();`. Then you can call `useIntl.formatMessage`.

For `formatMessage`, this takes an id of a string, so you can name it something like `ui-plugin-bursar-export.meta.settingTitle` or smth and then put the string you have right now in the `translations/en.json` file.

And then the `TitleManager` can just wrap the whole thing your returning and you don't need the `<> </>`.